docs/user-guide/sinks/user-defined-sinks.md

```diff
@@ -1,9 +1,19 @@
-# User Defined Sinks
+# User-defined Sinks
```

Comment:
Let's make it consistent, if `user-defined` is the term, then use it everywhere.

Reply:
Agree. I think `user-defined` is the right term. Will do.